packages/headless/src/features/query/query-analytics-actions.ts

```diff
@@ -10,10 +10,3 @@ export const logSearchboxSubmit = (): SearchAction =>
     AnalyticsType.Search,
     (client) => client.makeSearchboxSubmit()
   );
-
-export const logSearchboxAsYouType = (): SearchAction =>
```

Comment:
Isn't this a breaking change ? Was it exported globally ?

And someone could want to code an actual "search as you type" kind of experience (not instant result per say), and so that action would still be useful in this case I'd say.

Reply:
No it's past of the past PR I made yesterday and its not exported